README.md

```diff
@@ -26,6 +29,27 @@ will be just about your specific repository.
 - `metrics_files`: A list of relative paths to `metrics.yaml` files
 - `ping_files`: A list of relative paths to `pings.yaml` files
 
+There is also an `apps` list to define the concrete applications in the family.
```

Comment:
With regard to my point above, maybe we can start this with something like:

```suggestion
An application family consists of one or more applications built from the same code base. 

There is also an `apps` list to define the concrete applications in the family.
```

And then follow up with the Fenix example. That puts "application family" as a term in the text and explains it upfront and is only then followed by concrete examples.
(wording to be bikeshed)